app/model/ClientAsset.scala

```diff
@@ -118,3 +119,83 @@ object ClientAssetProcessing {
       status.failure.getOrElse(status.status)
   }
 }
+
+object ClientAssetMetadata {
+  implicit val format: Format[ClientAssetMetadata] = Jsonx.formatCaseClass[ClientAssetMetadata]
+}
+
+object ClientAssetTestData {
+  // SO MANY PERMUTATIONS! This helps test the UI without having to upload videos every time
+  def youTubeAsset(version: Long): ClientAsset = ClientAsset(
```

Comment:
feels odd to check this in...

Reply:
It's pretty useful to see the different types of result though (eg a find usages) as well as being useful for testing the UI.

I'd like to keep it if that's ok?